### ISSUE TITLE
Slight accessibility issue regarding image alt texts

### ISSUE THREAD

**Issue by RogeredBacon** [morkeltry/livinginthefuture#1]
Your images _robotboss.png_ and _avatar.png_ both lack alt text.
